docs/CONTRIBUTING/Dependencies.md

```diff
@@ -54,6 +54,27 @@ scl enable devtoolset-7 bash
 
 And build within the resulting bash shell, which has the tools first in PATH.
 
+Debian 9+ (stretch):
+```
+sudo apt-get install libx11-dev libx11-xcb-dev mesa-common-dev libgl1-mesa-dev libxcb-keysyms1-dev cmake python3-dev bison autoconf automake libpcre3-dev
+```
+
+For qrenderdoc:
+```
+sudo apt-get install qt5-qmake libqt5svg5-dev libqt5x11extras5-dev 
+```
+
+### Linux build
```

Comment:
I'd rather you remove this section please. There's already an example cmake invocation in the [documentation on compiling](https://github.com/baldurk/renderdoc/blob/v1.x/docs/CONTRIBUTING/Compiling.md), this file is the documentation on fetching build dependencies. This also rolls in distribution specific commands like setting `QT_SELECT` which might not apply everywhere.

Reply:
Got it, didn't realize there was a separate doc.